pages/join.js

```diff
@@ -21,6 +22,18 @@ function Join({ router }) {
     router.prefetch(profileUpdateURL);
   }, []);
 
+  function onChange() {
+    // I believe this is where the value gets sent to the API for the backend.
+    // value is the prop passed into this function.
+  }
+
+  const recaptchaReference = React.createRef();
+
+  const onSubmit = () => {
+    const { recaptchaValue } = recaptchaReference.current.getValue();
+    onSubmit(recaptchaValue);
```

Comment:
Wouldn't this be a recursive function?

Reply:
Yup... I’m not sure what the docs were trying to do. Let me look again when I go back on shift tonight. I am having a hell of a time working through this thing. It seems that we don’t even really need an onSubmit, but rather the onHandle could handle any changes?

Reply:
Ah, I see you were reading: https://github.com/dozoisch/react-google-recaptcha#component-instance-api

but `onSubmit` isn't the same thing as `this.props.onSubmit`

Notice how this component doesn't have onSubmit as a prop!

Reply:
Looking at their documentation: https://github.com/dozoisch/react-google-recaptcha#component-props

It looks like `onChange` fires when user successfully completes the captcha and `onErrored` fires when there's an issue.

Since this is just a crude attempt to scare scrapers, we can just do:

```
<ReCAPTCHA
  theme="dark"
  ref={ref}
  sitekey="6LcIHfMUAAAAANEzSejRpPilev7Hj0vI9fxc9q9D"
  onChange={() => {}}
  onErrored={() => {}}
/>
```